internal/runner/runner.go

```diff
@@ -90,13 +87,51 @@ func (r *Runner) RunEnumeration() {
 		}
 		return
 	}
+	// If the template path is glob
```

Comment:
So, this looks good to me. However, it breaks the previous way of things to work. @lc can you make the change backward compatible by making sure previous glob expressions work?

Reply:
Hey @Ice3man543, 

I'm not seeing how it's breaking the previous way of things working?

These glob expressions both work:

```
$ go run cmd/nuclei/main.go -t "../nuclei-templates/cves/*.yaml" -l ../targets.txt
$ go run cmd/nuclei/main.go -t "../nuclei-templates/cves/*" -l ../targets.txt
```

The code you highlighted checks if a user passed in a single YAML template 

```
if !strings.Contains(r.options.Templates, "*") && strings.HasSuffix(r.options.Templates, ".yaml") 
```